frontend/__tests__/components/utils/name-value-editor.spec.tsx

```diff
@@ -1,12 +1,22 @@
 import * as React from 'react';
 import { shallow } from 'enzyme';
-import { DragDropContext } from 'react-dnd';
+import { DndProvider } from 'react-dnd';
 import HTML5Backend from 'react-dnd-html5-backend';
 
 import { NameValueEditor } from '../../../public/components/utils/name-value-editor';
 
-describe(NameValueEditor.displayName, () => {
-  const Editor = DragDropContext(HTML5Backend)(NameValueEditor);
+const withDragDropContext = <TProps extends {}>(
```

Comment:
why do you declare this again and not just reuse the one from `drag-drop-context.tsx ` ?